internal/config/mysql_test.go

```diff
@@ -286,71 +291,174 @@ func TestMySQL_Opts(t *testing.T) {
 		if !errors.Is(err, w.err) {
 			return errors.Errorf("got_error: \"%#v\",\n\t\t\t\twant: \"%#v\"", err, w.err)
 		}
-		if !reflect.DeepEqual(got, w.want) {
-			return errors.Errorf("got: \"%#v\",\n\t\t\t\twant: \"%#v\"", got, w.want)
+		if !reflect.DeepEqual(len(got), len(w.want)) {
+			return errors.Errorf("length got: \"%#v\",\n\t\t\t\twant: \"%#v\"", len(got), len(w.want))
```

Comment:
```suggestion
			return errors.Errorf("length got: \"%#v\",\n\t\t\t\tlenght want: \"%#v\"", len(got), len(w.want))
```
if you'd like to output of length, you have to change like above.